packages/app-project/src/screens/ClassifyPage/components/RecentSubjects/RecentSubjects.spec.js

```diff
@@ -0,0 +1,16 @@
+import { shallow } from 'enzyme'
+import React from 'react'
+
+import RecentSubjects from './RecentSubjects'
+
+let wrapper
+
+describe('Component > RecentSubjects', function () {
+  before(function () {
+    wrapper = shallow(<RecentSubjects />)
+  })
+
+  it('should render without crashing', function () {
+    expect(wrapper).to.be.ok()
+  })
```

Comment:
At the risk of making this PR even longer, the `RecentSubjects` component needs tests.

Reply:
Separate PR?

Reply:
I created #1190 to track this.